docs/quickstart.rst

```diff
@@ -190,27 +190,25 @@ into ``models/dialogue``.
 ^^^^^^^^^^^^^^^^^^^^^^
 
 Now we can use that trained dialogue model to run our bot.
-We haven't included an NLU model yet, though, so we have to send 
-structured data to our bot directly. 
+We haven't included an NLU model yet, though, so we have to send
+structured data to our bot directly.
 
 You can play around with the bot, directly sending in the intents in the domain.
-To do this, start your message with a ``/``. 
+To do this, start your message with a ``/``.
 Give it a try by sending the message ``/greet``.
 
-If you are running these commands locally, run: 
+If you are running these commands locally, run:
 
 .. code-block:: bash
 
-   python -m rasa_core.run -d models/dialogue 
+   python -m rasa_core.run -d models/dialogue
 
 If you are running the cells here in the docs, run this cell:
 
 .. runnable::
 
-   from rasa_core.agent import Agent
-   from rasa_core.channels.console import ConsoleInputChannel
-   agent = Agent.load('models/dialogue')
-   agent.handle_channel(ConsoleInputChannel())
+   from rasa_core.run import start_server
+   start_server('models/dialogue')
```

Comment:
not entirely sure this is going to work from the docs, but couldn't test it either

Reply:
@amn41 do you know if we can start an http server here?

Reply:
yes you can! check out step 5 in http://rasa.com/docs/nlu/quickstart/

Reply:
nice alright, then we'll keep it and try it out :rocket: